packages/rn-tester-e2e/tests/screens/components.screen.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import {UtilsSingleton as Utils} from '../helpers/utils';
+
+// root level screen in RNTester: Components
+
+class ComponentsScreen {
+  // Reference in the top level Component list
+
+  buttonComponentLabelElement: string = Utils.platformSelect({
+    ios: '[label="Button Simple React Native button component."]',
+    android: '~Button Simple React Native button component.',
+  });
+
+  // Methods to interact with top level elements in the list
+
+  async checkButtonComponentIsDisplayed(): Promise<boolean> {
+    return await Utils.checkElementExistence(this.buttonComponentLabelElement);
+  }
+
+  async clickButtonComponent(): any {
+    await Utils.clickElement(this.buttonComponentLabelElement);
+  }
+}
+
+export const ComponentsScreenSingleton: ComponentsScreen =
```

Comment:
nit: If we are not keeping any state on these classes, maybe we could keep them as a plain object with constants/functions? It would avoid the need to rename these on import.

Reply:
Hi @kelset, 
This is fixed in the last MR. Could you please resolve this comment?